### DISCUSSION TITLE
Log Sources

### DISCUSSION THREAD

**Discussion by Steve5361** [Security-Onion-Solutions/securityonion#7171] [Unsupported Versions] [answered]
HI there, I wanted to check if SOS can use google authorization, AWS WAF, and AWS load balancers as log/data sources? I've have spent some time searching through the faq's without luck. Apologies if this is a repeated question. TIA

**Comment by dougburks** [comment, marked as answer]
The quick answer is: if it can send a log, then Security Onion most likely can consume the log.

The next question is about parsing the log. For that, I would check to see if there is an existing Filebeat module for the log:
https://docs.securityonion.net/en/2.3/filebeat.html#modules
https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-modules.html

If not, you might be able to develop your own ingest parser:
https://docs.securityonion.net/en/2.3/elasticsearch.html#parsing
https://www.elastic.co/guide/en/elasticsearch/reference/current/ingest.html
